app/components/elements/breadcrumb_component.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Elements
+  # Displays an item in the top bread crumb navigation
+  class BreadcrumbComponent < ApplicationComponent
+    def initialize(text:, link: nil, active: false)
+      @text = text
+      @link = link
+      @active = active
+      super()
+    end
+
+    attr_reader :text, :link
+
+    def truncated_title
+      truncate(text, length: 150, separator: ' ')
+    end
+
+    def classes
+      return 'breadcrumb-item active' if link.nil? || @active
+
+      'breadcrumb-item'
```

Comment:
I'm fuzzy on where we want to stick to `merge_classes`, but maybe @justinlittman can say more about whether we should be using that here. My two cents: ☝🏻 is perfectly readable.

Reply:
Breadcrumbs should be styled the same wherever they are, so I don't think we need to pass different classes to the component when calling it. It doesn't seem like we need merge_classes here (so far at least) since this is the only logic related to classes. But I could be missing something.